backend/Dockerfile.cloud_backend

```
@@ -8,6 +8,9 @@ ENV PYTHONPATH=/backend
 # 作業ディレクトリの設定
 WORKDIR /backend
 
+# FFmpegのインストール
+RUN apt-get update && apt-get install -y ffmpeg
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)